packages/devtools_app/lib/src/inspector/flutter/inspector_service_flutter_extension.dart

```diff
@@ -86,75 +88,107 @@ extension InspectorFlutterService on ObjectGroup {
     return await _evalUntilJsonIsNotEmpty(command);
   }
 
-  Future<RemoteDiagnosticsNode> getSummarySubtreeWithRenderObject(
+  Future<RemoteDiagnosticsNode> getLayoutExplorerNode(
     RemoteDiagnosticsNode node, {
     int subtreeDepth = 1,
   }) async {
     if (node == null) return null;
     final id = node.dartDiagnosticRef.id;
```

Comment:
we will want to avoid calling this method on VMs that are not new enough to safely define new VM service extensions like this.